addons/hr_holidays/models/hr_leave_allocation.py

```diff
@@ -214,6 +216,28 @@ def _update_accrual(self):
 
             holiday.write(values)
 
+    def _compute_accrual_plan(self):
+        accrual_plans=self.env['hr.accrual.plan'].search([("company_id","=",self.env.company.id)])
+        for record in self:
+            record.use_accrual_plan=False
+            contract_start=record.employee_id.contract_id.date_start or record.employee_id.contract_ids.date_start
+            # TO DO : level 
+            for accrual_plan in accrual_plans:
+                type = accrual_plan.start_after_type
+                count = accrual_plan.start_after_count
+                if type =='days':
+                    accrual_start = contract_start + relativedelta(days=+count)
+                if type =='months':
+                    accrual_start = contract_start + relativedelta(months=+count)
+                if type =='years':
+                    accrual_start = contract_start + relativedelta(years=+count)
```

Comment:
Actually, it's the same as: 
```suggestion
              accrual_start = contract_start + relativedelta(type=count)
```